src/AppBundle/Controller/ChatPageController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+class ChatPageController extends Controller
+{
+    public function pageAction()
+    {
+        return new Response(
+            $this->render('ChatPage.html.twig'));
```

Comment:
please name all views from a small letter and use low dashes to separate words, for example 'chat_page.html.twig',  also i think the word `page` is no need there.